src/components/atoms/NavItem/stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Story, Meta } from '@storybook/react';
+
+import NavItem, { NavItemProps } from '.';
+
+export default {
+  title: 'Components/Atoms/NavItem',
+  component: NavItem,
+} as Meta;
+
+export const Template: Story<NavItemProps> = (args) => <NavItem {...args}>{args.text}</NavItem>;
```

Comment:
You can remove the `export` of the Template

Reply:
Done